src/main/kotlin/no/nav/klage/oppgave/domain/joark/FagsaksSystem.kt

```diff
@@ -0,0 +1,20 @@
+package no.nav.klage.oppgave.domain.joark
+
```

Comment:
Denne burde inneholde info om system. Ikke så lett å se om vi har dekt alle systemene basert på koder alene.

Reply:
Usikker på hva du mener her. Lista er basert på det dokumentasjonen på grensesnittet oppgir som lovlige verdier: https://confluence.adeo.no/display/BOA/opprettJournalpost
Det gjelder jo alt av enumer her, tenker at det ikke er noen koblinger som garanterer for utfyllende liste utover det som er oppgitt på den siden.